docs/code-of-conduct-response.rst

```diff
@@ -1,76 +1,121 @@
 
-* A third offense that resulted in a warning from staff
-* Continuing to harass after any "No" or "Stop" instruction
-* A pattern of harassing behavior, with or without warnings
-* A single serious offense (e.g., punching or groping someone)
-* A single obviously intentional offense (e.g., taking up-skirt photos)
+When receiving a report, the response team acknowledges receipt as soon as possible, and aims to be understanding and compassionate. However, no commitment should be made on whether this is a violation or which action will be taken.
+
+If the reporter desires it, arrange for an escort by conference staff or a trusted person, contact a friend, or contact local law enforcement. Do not pressure the reporter to take any action if they do not want to do it. Respect the reporter's privacy by not sharing details with others.
+
+Acting as a team
+~~~~~~~~~~~~~~~~
+
+The response team generally works, decides and communicates as a team. If the report indicates that immediate action is required and other response team members are not available, any response team member may take the action they think is necessary. If possible, it’s still preferred to take action after a brief discussion with at least one other response team member, or even any other organiser, rather than acting entirely unilaterally. Formal processes and guidelines must never get in the way of preventing threats to anyone’s safety.
+
+For example, you may consider unilateral action:
+
+* When any delay of action will likely bring unacceptable further harm to others. Ongoing aggression or serious harassment are examples of this.
+* If you feel people are in immediate physical danger, it can be appropriate to contact local law enforcement.
+* If you witness a presentation which is clearly in violation of the Code of Conduct repeatedly or very seriously. You might simply say “I’m sorry, this presentation cannot be continued at the present time”. Examples of when this is justified could be significant threats of violence, harassment of others, or continuous sexist jokes. Do not end a presentation in cases like a few inappropriate jokes - in that case do report it to the response team.
+
+Reviewing the report
+~~~~~~~~~~~~~~~~~~~~
+
+Based on the report and any other available information, the response team will meet to determine, to the best of their ability:
+
+* What happened
+* Whether this event constitutes a Code of Conduct violation
+* Who, if anyone, was the bad actor(s)
+* What the appropriate resolution is
+
+A written record will be kept of each incident and its review. Although that sometimes seems superflous at the time, hectic environments like conferences can make it easy to forget or confuse details later.
```

Comment:
Should we set up a CoC Trello board for the team (visible only to members), where we can track new/WIP/closed incidents?

Reply:
Yes. Or a google sheet, that's what we use for Django. This also requires some policy, of which I've written things for Django that we can reuse.